.travis.yml

```diff
@@ -90,10 +90,16 @@ matrix:
         - CPPFLAGS=-DXXH_VECTOR=4 CFLAGS="-O3 -maltivec -mvsx -mpower8-vector -mcpu=power8" LDFLAGS="-static" make check
 
     - name: IBM s390x compilation and consistency checks
+      dist: bionic
       arch: s390x
       script:
+        # for info
+        - cat /proc/cpuinfo
```

Comment:
good idea ! :+1:

Reply:
Some reason it doesn't output it though…

Reply:
If I am not mistaken, adding this to the root of the file will apply to all jobs and fold at the start, right?

```yml
before_install:
   - cat /proc/cpuinfo
```

Reply:
Seems so.
Never used it though...